bokehjs/test/unit/models/tools/inspectors/hover_tool.ts

```diff
@@ -74,6 +76,32 @@ describe("HoverTool", () => {
     })
   })
 
+  describe("line_policy", () => {
+    it("should support 'none' line_policy with mode=vline", async () => {
+      const tooltips: [string, string][] = [["x", "$x"], ["y", "$y"]]
+      const hover = new HoverTool({mode: "vline", line_policy: "none", tooltips})
+      const p = fig([200, 200], {toolbar_location: null, min_border: 0, x_range: [1, 3], y_range: [0, 2]})
+      const r = p.line([1, 2, 3], [1, 1, 1])
+      p.add_tools(hover)
+
+      const {view} = await display(p)
+
+      const crv = view.renderer_views.get(r)!
+      const [[sx], [sy]] = crv.coordinates.map_to_screen([1.8], [1.5])
+      const ui = view.canvas_view.ui_event_bus
+      const {left, top} = offset_bbox(ui.hit_area)
+      const ev = new MouseEvent("mousemove", {clientX: left + sx, clientY: top + sy})
+      ui.hit_area.dispatchEvent(ev)
+
+      await view.ready
+
+      const hover_view = view.tool_views.get(hover)! as HoverTool["__view_type__"]
+      const tt = hover_view.ttmodels.values().next().value as Tooltip
+
+      expect(tt.position).to.be.equal([sx|0, sy|0])
```

Comment:
I'm assuming based on vague recollection that this truncation is what happens internally. If there are better comparisons to make, I am open to suggestion.

Reply:
`map_to_screen()` produces `Float32Array` so truncation of `sx` and `sy` shouldn't be necessary. If anything, it's `position` it should be truncated. I need to look closer into this.

Reply:
> I need to look closer into 

Do you mean in the future, or in the context of this PR? It would be helpful for me in cases like this if you would speak explicitly to the open decision: are you asking that merging this PR be delayed?

Reply:
I want to do it here. Hopefully it won't take too long.

Reply:
So it's actually not guaranteed that `tt.position` is whole pixels, thus both sides should be truncated, although in this particular setup `tt.position` is pixels. I changed that and converted this test into visual regression test.